### PR TITLE
Collect decommissioned servers

### PR DESCRIPTION
**Pull request by gberginc**
This commit collects decommissioned servers from the available resource
pool and makes them available for the recommission operation performed
by the end user.

### CODE REVIEW

**Comment by gberginc** [comment]
Thanks, @tjazsch for updating the specs, much appreciated!

@agrare this patch here adds a capability to collect basic information about the servers that are only available in the pool for users to recommission and later use. Without a server being recommissioned, it is of no use and thus also very limited in the data that it provides. So far, we therefore just get the information about the availability of an unused resource. We will be investigating the APIs to see if we can at least get some more info, but for the time being, for the end-user workflow, we need references to unused servers.

File: app/models/manageiq/providers/cisco_intersight/inventory/parser/physical_infra_manager.rb [hunk 1]
```diff
@@ -11,7 +11,7 @@ def parse
     # Methods that are executed during refresh call
 
     def physical_servers
-      # Parses physical servers and its details
+      # Parsing active/undecomissioned servers:
```
Comment (agrare) [inline app/models/manageiq/providers/cisco_intersight/inventory/parser/physical_infra_manager.rb:14]: Don't believe `undecomissioned` is a word, might just `Parsing active/ servers` work or `active/non-decommissioned` ?

**Comment by agrare** [comment]
Looks great just the one comment

File: app/models/manageiq/providers/cisco_intersight/inventory/parser/physical_infra_manager.rb (hunk 1 above)
Comment (tjazsch) [reply app/models/manageiq/providers/cisco_intersight/inventory/parser/physical_infra_manager.rb:14]: True. Just writing "parsing active servers" with the next commit.

**Review by agrare — APPROVED** [review]
:+1: LGTM

**Pull request merged**